frontend/src/lib/UriUtil.ts

```diff
@@ -96,11 +97,24 @@ function isHttps(): boolean {
   return window.location.href.startsWith("https://")
 }
 
+/**
+ * Run SVG strings through DOMPurify to prevent Javascript execution
+ */
+function sanitizeSvg(uri: string): string {
+  const SVG_PREFIX = "data:image/svg+xml;utf8,"
```

Comment:
I would drop the `utf8` part. LMK what you think.

Reply:
Good idea, I didn't realize it was unnecessary. Thanks!